### DISCUSSION TITLE
15 settembre 2022 - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#19] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/ee6f8a59-0f45-4df3-8f49-483048024807)

**Comment by niccolo-fato** [comment]
Non sono sicuro sia corretto, nel caso fatemi sapere se lo è
![2024-01-13 15 35 58](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/6ef07184-cbe1-4cf6-a9ca-b337d8c97711)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/a45f3c19-0c45-4b00-a8db-5fbbba4f4703)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/2b936554-e87a-4c18-bade-693aef2a13d9)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-22 153343](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/1073208e-4bda-47d7-8139-8abf6cbfc798)
![Immagine 2024-01-22 153356](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/084f6baf-95ca-43ef-a6e2-3fa5115d1472)
![Immagine 2024-01-22 153407](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/e3a768b7-cfcd-4927-8614-6b98b450e893)
